src/source/image_source.js

```diff
@@ -78,7 +79,7 @@ class ImageSource extends Evented implements Source {
     dispatcher: Dispatcher;
     map: Map;
     texture: Texture | null;
-    image: HTMLImageElement | ImageBitmap;
+    image: ImageData;
```

Comment:
Question: Makes perfect sense. Just out of curiosity, as far as flow goes, is `ImageData | null` just not worth the trouble in this case?

Reply:
I just didn't change the status quo in this particular PR, but we do need to make a sweep to clean this up — sloppy typing like this is everywhere both in this class and all over the codebase, and Flow isn't complaining. :(